SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -58,24 +59,599 @@ void CDriver::PreprocessPythonInterface(CConfig** config, CGeometry**** geometry
   }
 }
 
+/////////////////////////////////////////////////////////////////////////////
+/* Functions related to the far-field flow variables.                      */
```

Comment:
Looks like you have weird symbols here

Reply:
Hmm that's strange, I don't see the weird symbols... but I rewrote these few lines so hopefully they're gone.